deploy/roles/stata/tasks/main.yaml

```diff
@@ -157,6 +157,11 @@
   tags:
     - stata
 
+- name: DEBUG - see what is in stata_install_directory
```

Comment:
FYI, the install directory only exists in our real environment, since we don't have a local installer for Stata to use due to licesning.

Reply:
Thanks.  I've removed the DEBUG task.